extension.json

```diff
@@ -1,30 +1,21 @@
 {
   "name": {
-    "en": "Name of extension (required in at least one language)"
+    "en": "Amendment rationale classifications"
   },
   "description": {
-    "en": "A brief description of what this extension does, this will be pulled into the main docs as a link (required in at least one language)"
+    "en": "Adds a field to the amendment object to classify the rationale for the amendment."
   },
   "documentationUrl": {
-    "en": "http://httpbin.org/anything/path/to/documentation/in/specified/language"
+    "en": "https://github.com/open-contracting-extensions/ocds_amendment_rationale_classifications_extension"
   },
   "schemas": [
-    "record-package-schema.json",
-    "release-package-schema.json",
     "release-schema.json"
   ],
-  "codelists": [
-    "emptyCodelist.csv"
```

Comment:
Good spot! Based on https://extensions.open-contracting.org/en/extensions/procurementMethodRationaleClassifications/master/codelists/#+itemClassificationScheme.csv I think category should be 'amendmentRationaleClassifications' since there could be other ways to classify amendments. I also namespaced the code with 'eu-' as suggested elsewhere.